lib/marked.cjs

```diff
@@ -2024,23 +2027,27 @@ var Marked = class {
       if (pack.renderer) {
         const renderer = this.defaults.renderer || new _Renderer(this.defaults);
         for (const prop in pack.renderer) {
-          const prevRenderer = renderer[prop];
-          renderer[prop] = (...args2) => {
-            let ret = pack.renderer[prop].apply(renderer, args2);
+          const rendererFunc = pack.renderer[prop];
+          const rendererKey = prop;
```

Comment:
Is the variable for rendererKey needed? Can you use prop instead?

Reply:
This is so Typescript knows what type prop is. Without this it is a string and renderer[prop] is `any` (implicitly) This is a generated file. in the .ts file it is `const rendererKey = prop as  keyof _Renderer;`

Reply:
Oops, I keep looking at the wrong files.

I think there is a way to tell github which files are generated so they start out collapsed in code reviews 🕵️‍♂️

Reply:
Added in PR #2908